lib/http/connect.js

```diff
@@ -119,6 +122,10 @@ p.getHandler = function() {
         });
       }, $this.options.stepTimeout)],
       optimizedOriginalImage: ['requestedImage', async.timeout(function optimizedOriginalImage(results, cb) {
+        if (reqInfo.useOriginal) {
```

Comment:
should be: `reqInfo.useOriginal || reqInfo.urlInfo.query.useOriginal === 'true'`

to account for cases of no image steps use of original (where you're forwarding content type), and when query param useOriginal is specified and there are image steps. The idea is that you can still perform image operations on the original, when desired.